packages/docs/content/foundation/typography/headings.mdx

```diff
@@ -197,8 +199,12 @@ The following CSS variables can be overridden to change the type sizes:
 - `--font-size-md`, uses `--font-size-base`
 - `--font-size-sm`
 
+<ThemeContent onlyThemes={['medicare', 'healthcare']}>
```

Comment:
guidance only exists for these themes, so hide this heading unless these themes are active